src/components/widgets/see-more-button/SeeMoreButton.tsx

```diff
@@ -0,0 +1,17 @@
+'use client';
+
+import Link from 'next/link';
+
+export function SeeMoreButton() {
+  return (
+    <div className="flex justify-center">
```

Comment:
Maybe this `div` is not needed, see if you can move the styles directly to the `Link` component.